checker/node/examples.py

```diff
@@ -639,6 +639,7 @@
     'NOAA/NOAA_VIIRS_DNB_MONTHLY_V1_VCMCFG',
     'NOAA/NOAA_VIIRS_DNB_MONTHLY_V1_VCMSLCFG',
     'NRCan/NRCan_CDEM',
+    'NRCan_CFS_LFC/NRCan_CFS_LFC_SCANFI',
```

Comment:
could you remove this? The public asset id will be projects/gcpm041u-lemur/assets/scanfi_v12/SCANFI_v1_2, so you don't need this exception.